Documentation/README_MRTKStandardShader.md

```diff
@@ -23,6 +23,21 @@ A custom material inspector exists for the MRTK/Standard shader called **MixedRe
 
 ![materialinspector](../Documentation/Images/MRTKStandardShader/MRTK_MaterialInspector.jpg)
 
+## Performance
+
+One of the primary advantages to using the MRTK Standard shader over the Unity standard shader is performance. The MRTK Standard Shader is extensible to only utilize the features enabled. However, the MRTK Standard shader has also been written to deliver comparable aesthetic results as the Unity Standard shader but at a much lower cost. One simple way to compare shader performance is via the number of operations that needs to be performed on the GPU. Of course, the magnitude of calculations may fluctuate by features enabled and other rendering configurations. But, in general, the MRTK Standard shader performs significantly less computation than the Unity Standard shader.
+
+Unity Standard shader statistics example
+
+![Unity Standard Shader Statistics](../Documentation/Images/Performance/UnityStandardShader-Stats.png)
+
+MRTK Standard shader statistics example
+
+![MRTK Standard Shader Statistics](../Documentation/Images/Performance/MRTKStandardShader-Stats.png)
```

Comment:
Did you need to commit this image too?

Reply:
Yes, it is already there under the performance page

Reply:
Weird, GitHub is failing to render the image for me.

![image](https://user-images.githubusercontent.com/13305729/61569181-05496800-aa3b-11e9-9b97-2c869140d0a1.png)

Looks like the only difference is the case in the extension? .PNG vs .png

Reply:
It also fails on the performance page....you have to go to github.io for it to work:
https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Performance/PerfGettingStarted.html

![image](https://user-images.githubusercontent.com/25975362/61569376-e8616480-aa3b-11e9-8396-13964ca627ad.png)

Reply:
This lead me down [a rabbit hole](https://stackoverflow.com/questions/7996919/should-url-be-case-sensitive) haha.

Do both of these links work for you?

1) https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Documentation/Images/Performance/UnityStandardShader-Stats.png

2) https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Documentation/Images/Performance/UnityStandardShader-Stats.PNG

Reply:
The second one (.PNG) works for me. .png with lowercase is showing 404 error.

Reply:
@cre8ivepark thank you for testing! @Troy-Ferrell I would recommend switching to the case on disk, so .PNG

Reply:
interesting, it must not matter with markdown on github.io but on github.com it matters. I'll just change it to be holistic. Thanks guys!